File: cfgs/pythoncfgs/SEARCH_INSERT_AND_DELETE_IN_A_SORTED_ARRAY.py
Nodes
1;conditional;foo_1 < foo_2;8;
2;statement;foo_1 = (foo_2 + foo_3) / 2;10;
3;conditional;foo_1 == foo_2[fun_1(foo_3)];11;
4;conditional;foo_1 > foo_2[fun_1(foo_3)];13;
5;statement;return fun_1(foo_1, foo_2, foo_3 - 1, foo_4);15;
6;statement;return fun_1(foo_1, foo_2 + 1, foo_3, foo_4);14;
7;statement;return foo_1;12;
8;statement;return -1;9;
9;exit;;
Edges
0;2;8;
1;3;
2;4;7;
3;5;6;
4;9;
5;9;
6;9;
7;9;
8;

File: cfgs/java_cfgs_foo/SEARCH_INSERT_AND_DELETE_IN_A_SORTED_ARRAY.java
Nodes
1;special;;;
2;conditional;foo_1 < foo_2;14;
3;statement;return -1;14;
4;exit;;;
5;statement;int mid = (foo_1 + foo_2) / 2;15;
6;conditional;foo_1 == foo_2[foo_3];16;
7;statement;return foo_1;16;
8;conditional;foo_1 > foo_2[foo_3];17;
9;statement;return fun_1(foo_1, foo_2 + 1, foo_3, foo_4);17;
10;statement;return fun_1(foo_1, foo_2, foo_3 - 1, foo_4);18;
Edges
0;2;
1;3;5;
2;4;
3;
4;6;
5;7;8;
6;4;
7;9;10;
8;4;
9;4;
